### DISCUSSION TITLE
coin combine command finishes, but never does anything..

### DISCUSSION THREAD

**Discussion by 172pilot** [Chia-Network/chia-blockchain#18954] [Support]
I was trying to run "chia wallet coins combine" to combine a lot of tiny transactions from being in a pool.  The transaction goes through and I can see it say it completes, but the number of coins never goes down.  I am up to over 1400 partial coins.  I have run the command 5 or 6 times over the last month with the same results.  How can I get this to work?  Thanks in advance!!

**Comment by 172pilot** [comment]
One potential correction..  I am not SURE that the transaction completes.  I can see it as pending, and eventually it goes away, but I dont see it in the output for chia wallet get_transactions after it is no longer pending, while I can see other pool transactions remaining in the list.  Is it possible it's timing out and just being removed from the list without completing, and if so, is there a way to diagnose?

**Comment by 172pilot** [comment]
I am seeing other posts, where people are having the same problems with the coins combine command.  Is this just a known bug in the current CHIA executables?  I can't find that anyone has solved the problem.

**Comment by BrandtH22** [comment]
Hey @172pilot , as a reminder we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)

As for the coin combine command, make sure you are adding the wallet id flag (-id) and also the coin number flag (--number_of_coins). 

Also if you would prefer a GUI based method of combining coins you can try out the in development Sage wallet: https://github.com/xch-dev/sage/releases/tag/v0.8.1

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
